modular_skyrat/modules/modular_weapons/code/melee.dm

```diff
@@ -23,6 +23,7 @@
 	righthand_file = 'modular_skyrat/modules/modular_weapons/icons/mob/inhands/weapons/swords_righthand.dmi'
 	block_chance = 20
 	armour_penetration = 25
+	force = 15
```

Comment:
Maybe make it moreso a defensive weapon than a offensive weapon to buff it? 30% block chance? Either that or slow down it's attack speed instead